tests/testthat/test-user_utils.R

```diff
@@ -19,6 +19,7 @@ test_that("convert_blanks_to_na Test 2: attributes are preserved when converting
 
 ## Test 3: blank strings are turned into `NA` inside data frames ----
 test_that("convert_blanks_to_na Test 3: blank strings are turned into `NA` inside data frames", {
+  # nolint start: undesirable_function_linter
```

Comment:
Can we redo these tibbles like we do in other tests and drop this use of structure?  then we can drop a few of these `nolints` ??

Reply:
For the tests, I have changed from nolint around structure) to create the tibbles as is done in other tests.